packages/page-accounts/src/Accounts/index.tsx

```diff
@@ -347,6 +352,11 @@ function Overview ({ className = '', onStatusChange }: Props): React.ReactElemen
               />
             </>
           )}
+          <Button
+            icon='sign-in-alt'
+            label={t<string>('From Snap')}
+            onClick={connectSnap}
```

Comment:
Clicking first time installs the snap. I get this, not sure if expected, info:
![Screenshot from 2024-01-19 10-20-07](https://github.com/Cardinal-Cryptography/azero-dev/assets/35614528/5a2d835d-ac41-4292-a6ad-df8a61152d2f)
but account isn't connected.
Clicking it again does nothing, but shows error in console:
![image](https://github.com/Cardinal-Cryptography/azero-dev/assets/35614528/60c51c77-911c-4db7-8c62-c9c9cdc0cb47)

Reply:
This is an issue caused by a recent Flask release, i.e. a breaking change for the `azero-wallet` snap. I will fix it and get back you. In the meantime, I will address other comments in the PR.

Reply:
This is an issue with the snap itself, pleas see [this comment](https://github.com/Cardinal-Cryptography/azero-dev/pull/95#discussion_r1460925687).

Reply:
I deployed a fix. You should be able to test it locally now using the latest commit.